time/src/test/java/net/jqwik/time/api/YearTests.java

```diff
@@ -45,6 +46,27 @@ void yearIsNotZero(@ForAll("yearsAround0") Year year) {
 
 	}
 
+	@Group
+	class validYearsAreGeneratedWithAnnotation {
```

Comment:
jqwik convention is to have Group classes be upper case. Use `@Label("myname")` if you want the displayed test name to be different.